test/e2e/settings.test.js

```diff
@@ -1,4 +1,15 @@
-import { test } from './setup/coverage.js'
+import { readFile } from 'node:fs/promises'
+
+import { test, expect } from './setup/coverage.js'
+
+const languageFilePromise = readFile('./src/lib/languages.json', 'utf8')
+
+let languages
+const getLanguages = async () => {
+  if (languages != null) return languages
+  languages = JSON.parse(await languageFilePromise)
+  return languages
+}
```

Comment:
playwright didn't like me importing json and i didn't want to fiddle with the build since this PR is big enough.